sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/SigningCertificatesAsyncClient.java

```diff
@@ -30,26 +32,23 @@
     /**
      * Retrieves metadata signing certificates in use by the attestation service.
      *
-     * @throws CloudErrorException thrown if the request is rejected by server.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @param context - Context to associate with this operation.
      * @return the response.
      */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<JsonWebKeySet>> getWithResponse() {
-        return this.serviceClient.getWithResponseAsync()
+//    @ServiceMethod(returns = ReturnType.SINGLE)
```

Comment:
Is this intentional?

Reply:
I'm about to delete the file, so I'm not sure it matters :).